### ISSUE TITLE
Support square styling of unordered list in CMS help text

### ISSUE THREAD

**Issue by davidmpickett** [department-of-veterans-affairs/va.gov-cms#17170]
## User Story or Problem Statement

As a Product Team building on the Drupal CMS, I need unordered lists in Help Text to use the square style of bullet
so I can avoid having screen reader users have a negative experience.

## Context

While updating the help text for the VBA Facility content type, we ran into a situation where it is semantically accurate to have an unordered list. However, the unordered list cannot be styled with square bullets like the VA.gov default.

![Screenshot 2024-02-08 at 12 46 36 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/51967950/f56e0115-7c40-428f-a7b1-a2590766c098)
c703db0aa2ec)

From @laflannery:

> Screen readers will reader out the default list item styles (the black circles) as "bullet" and that type of language could potentially be triggering for our user base. This is why our default VA.gov list item style is a black square, and it is announced as such.

From @omahane: 

> I added that, but the style setting for square list markers is stripped, as the Help text supports a subset of tags and not "style" attribute.

## Acceptance Criteria
- [ ] Testable_Outcome_X
- [ ] Testable_Outcome_Y
- [ ] Testable_Outcome_Z
- [ ] Requires Content Model Documentation
- [ ] Requires design review
- [ ] Requires KB article update

## Implementation steps
<optional>

**Comment by davidmpickett** [comment]
@omahane @laflannery - I spun this off of the PR review. Let me know if this is accurately documented.

**Comment by laflannery** [comment]
The ticket has been updated to reflect the suggested update to the CMS design system
